plugins/auth-backend/src/service/router.ts

```diff
@@ -249,3 +249,21 @@ export function createOriginFilter(
     return allowedOriginPatterns.some(pattern => pattern.match(origin));
   };
 }
+
+export function getDefaultBackstageTokenExpiryTime(config: Config) {
```

Comment:
As you can see in the build error logs, this function accidentally got exported all the way to the public top of the package interface, because the root `index.ts` exports `*` from this file. Could you adjust that file so that it exports everything except this function? The end goal is that you run `yarn build:api-reports` and the api-report.md file of this package is unchanged compared to master.